src/client/visualizations/line-chart/line-chart.tsx

```diff
@@ -669,6 +671,7 @@ export class LineChart extends BaseVisualization<LineChartState> {
     const { axisRange, scaleX, xTicks } = this.state;
     const { splits, timezone } = essence;
 
+    const showLegend = splits.length() === 2;
```

Comment:
Is `2` a magic number or something like `MIN_SPLITS_COUNT_TO_SHOW_THE_LEGEND`?

Reply:
It's an exact number of splits when to show legend. I don't think we gain anything from condition that read "show legend only if there are exactly that many splits that should be present to show legend".

Maybe we could invent some name for "second split". Something like **nested split** or **non-continuous split** ... Then we could have function `hasNestedSplit` and write `const showLegend = hasNestedSplit(essence)` 🤔

Reply:
Will do in #602